components/github_alert.lua

```diff
@@ -0,0 +1,89 @@
+local util = require 'util'
+local json = require 'json'
+local http_request = require 'http.request'
+local os = require 'os'
+local io = require 'io'
+local luadate = require 'date'
+commit = false
+commit_of_day = 0
+first_time = true
```

Comment:
Why global variables here?

And keeping all custom variable go behind all modules is recommended. Here the custom variable is followed by another module `cairo`